lbrynet/core/Error.py

```diff
@@ -84,6 +84,12 @@ class NoSuchStreamHashError(Exception):
     pass
 
 
+class NoSuchSDHashError(Exception):
```

Comment:
a pet peeve of mine: its not necessary to have `Error` in your exception.  Its redundant because its always obvious in context that you're dealing with an error.. Ie - `raise NoSuchSDHash(...)` is enough.